File: Leetcode/1985. Find the Kth Largest Integer in the Array.cpp
class Solution {
public:
    string kthLargestNumber(vector<string>& nums, int k) {
        nth_element(nums.begin(), nums.begin()+k-1, nums.end(), [](string s1, string s2) {
            return s1.length()==s2.length()?s1>s2:s1.length()>s2.length();
        });
        return nums[k-1];
            
    }
};

File: Leetcode/1985. Find the Kth Largest Integer in the Array.py
class Solution(object):
    def kthLargestNumber(self, nums, k):
        """
        :type nums: List[str]
        :type k: int
        :rtype: str
        """
        arr=[ int(i) for i in nums]
        arr.sort();
        return str(arr[len(arr)-k])
        
